### PR TITLE
fix: order contributors according to DESCRIPTION author list

### PR DESCRIPTION
**Pull request by anirbanshaw24**
*Have you read the [Contributing Guidelines](https://github.com/Appsilon/.github/blob/main/CONTRIBUTING.md)?*

Issue #
Correct contributors of packages and their order.

## Changes description

*Clearly and concisely describe what's in this pull request. Include screenshots, if necessary.*

### CODE REVIEW

File: app/scripts/contributors.js [hunk 1]
```diff
@@ -9,102 +9,88 @@ export const addContributors = () => {
...
-      {'img': '10000651', 'profile': 'kamilzyla', 'credit': 'Creator & Maintainer'},
-      {'img': '7279715', 'profile': 'jakubnowicki', 'credit': 'Contributor'},
-      {'img': '38053499', 'profile': 'TymekDev', 'credit': 'Contributor'},
-      {'img': '1421503', 'profile': 'marekrogala', 'credit': 'Contributor'}
+      {'img': '10000651', 'profile': 'kamilzyla', 'credit': 'Author & Maintainer'},
+      {'img': '7279715', 'profile': 'jakubnowicki', 'credit': 'Author'},
+      {'img': '32634801', 'profile': 'Leszek-Sieminski', 'credit': 'Author'},
+      {'img': '1421503', 'profile': 'marekrogala', 'credit': 'Author'},
+      {'img': '110383037', 'profile': 'vibalre', 'credit': 'Author'},
+      {'img': '103684742', 'profile': 'tmakowski', 'credit': 'Author'},
```
Comment (jakubnowicki) [inline app/scripts/contributors.js:72]: Please replace with [https://github.com/TymekDev](https://github.com/TymekDev)

**Review by jakubnowicki — CHANGES_REQUESTED** [review]
Overall looks good. I have one request: Please replace GitHub more contributors links with the ones pointing to documentation pages, e.g. for Rhino use https://appsilon.github.io/rhino/authors.html

File: app/scripts/contributors.js (hunk 1 above)
Comment (anirbanshaw24) [reply app/scripts/contributors.js:72]: Ahh okay.

**Review by jakubnowicki — APPROVED** [review]
LGTM

**Pull request merged**